Sources/DeepLearning/Layers/Core.swift

```diff
@@ -0,0 +1,229 @@
+// Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#if !COMPILING_TENSORFLOW_MODULE
+@_exported import TensorFlow
```

Comment:
One last request: Could you please remove `@_exported` from `import` declarations in files inside `Layers/`?

Reply:
This is because `TensorFlow` has already been re-exported in the `DeepLearning/Layer.swift` file.

Reply:
yea alright.

Reply:
@rxwei should i remove it from Layer.swift as well?